core/src/main/java/tech/pegasys/web3signer/core/metrics/Web3SignerMetricCategory.java

```diff
@@ -34,6 +36,7 @@
         ImmutableSet.<MetricCategory>builder()
             .addAll(EnumSet.allOf(Web3SignerMetricCategory.class))
             .addAll(EnumSet.allOf(StandardMetricCategory.class))
+            .addAll(EnumSet.allOf(SlashingMetricCategory.class))
```

Comment:
Not sure if it matters, but this would mean the SlashingMetricCategory would be available for Eth1 and Filecoin subcommands even though the metric category can never be used. Should we make this conditional on the subcommand?

Reply:
That _sounds_ kind hard to organise